readme-vars.yml

```diff
@@ -25,11 +40,16 @@ param_env_vars:
 param_usage_include_ports: false
 param_usage_include_net: true
 param_net: "host"
-param_net_desc: "Shares host networking with container. Required for some devices to be discovered by Home Assistant."
+param_net_desc: "Shares host networking with container."
 param_device_map: true
 param_devices:
   - { device_path: "/path/to/device", device_host_path: "/path/to/device", desc: "For passing through USB, serial or gpio devices." }
 
+# optional container parameters
+opt_param_usage_include_ports: true
+opt_param_ports:
+  - { external_port: "8123", internal_port: "8123", port_desc: "Application WebUI, only use this if you are not using host mode." }
```

Comment:
Please change the text to something beginning with optional, optionally, #optional or something similar so it looks like 
"Optionally assign the application webUI port, when not using host mode" or something. you'll notice our other optional fields look similar to what im saying.

Reply:
>you'll notice our other optional fields look similar to what im saying

But you don't conform to that wording in 89% of the [other READMEs](https://github.com/search?l=&p=1&q=org%3Alinuxserver+opt_param_ports+filename%3Areadme-vars.yml&ref=advsearch&type=Code)

Reply:
This is fine, our templating marks it as optional